bin/dist

```diff
@@ -6,7 +6,7 @@ REV=$(git rev-parse --short HEAD)
 
 CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -ldflags "-X main.rev=$REV"
 
-OWNER=nytimes
+OWNER=${OWNER-nytimes}
```

Comment:
This is the docker hub org owner: https://hub.docker.com/u/nytimes/

A developer who wants to push it to their own registry can specify `OWNER=me ./bin/dist`, otherwise `-nytimes` sets the default variable value.